fdbserver/TagPartitionedLogSystem.actor.cpp

```diff
@@ -33,6 +33,17 @@
 #include <fdbserver/ptxn/TLogPeekCursor.actor.h>
 #include "flow/actorcompiler.h" // This must be the last #include.
 
+ACTOR Future<Version> minVersionWhenReady(Future<Void> f, std::vector<Future<ptxn::TLogCommitReply>> replies) {
```

Comment:
This could be defined a template function with either `ptxn::TLogCommitReply` or `TLogCommitReply` as some `class T`, i.e.,

```
ACTOR template <class T>
Future<Version> minVersionWhenReady(Future<Void> f, std::vector<Future<T>> replies) {
```

Thus, only one `minVersionWhenReady` is needed.